src/Workspaces/Remote/Core/Shared/Extensions.cs

```diff
@@ -58,7 +58,7 @@ private static async Task AppendAssetMapAsync(this Project project, Dictionary<C
 
             projectChecksums.Find(project.State, Flatten(projectChecksums), map, cancellationToken);
 
-            foreach (var document in project.Documents.Concat(project.AdditionalDocuments))
+            foreach (var document in project.Documents.Concat(project.AdditionalDocuments).Concat(project.AnalyzerConfigDocuments))
```

Comment:
so test probably worked even with the issue above since it explicitly added AnalyzerConfigDocuments here.